frontend/src/pages/SendMetadata/MetadataForm.jsx

```diff
@@ -0,0 +1,205 @@
+  align-content: space-around;
+`;
+
+const Input = styled.input`
+  flex: 0 0 2em;
+  margin: 0.5em 1em;
+`;
+
+const Select = styled.select`
+  flex: 0 0 2em;
+  margin: 0.5em 1em;
+`;
+
+const TextArea = styled.textarea`
+  flex: 0 0 10em;
+  margin: 0.5em 1em;
+`;
+
+const RadioLabel = styled.label`
+  border-radius: 0.2em;
+  padding: 0.2em;
+  ${({ background }) => (background ? `background-color: ${background};` : '')}
+  ${({ border }) => (border ? `border: 1px solid ${border};` : '')}
+`;
+
+const HorizontalWrapper = styled.div`
+  flex: 0;
+  display: flex;
+  flex-direction: row;
+`;
+
+export const MetadataForm = () => {
+  const [state, setState] = useState({
+    metadataTypeName: '',
+    releaseState: 1,
+    description: '',
+    formatName: '',
+    municipalityName: '',
+    url: '',
+  });
+
+  // municipalities should be objects with a "name" key
+  const [municipalities, setMunicipalities] = useState([]);
+  const [metadataTypes, setMetadataTypes] = useState([]);
+  const dataFormats = ['JSON', 'CSV'];
+
+  const [submissionStatus, setSubmissionStatus] = useState('');
+
+  const dispatch = useDispatch();
+
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+    const nextState = { ...state };
+    nextState[name] = value;
+    setState(nextState);
+  };
+
+  const handleRadioChange = (event) => {
+    const { name, value } = event.currentTarget;
+    const nextState = { ...state };
+    nextState[name] = parseInt(value, 10);
+    setState(nextState);
+  };
+
+  // fetch municipalities
+  useEffect(() => {
+    const internal = async () => {
+      try {
+        const res = await fetch('/api/Municipality');
+        const { status, ok } = res;
+        if (!ok) {
+          const err = new Error();
+          err.status = status;
+          throw err;
+        }
+        const receivedMuniciplalities = await res.json();
+        if (receivedMuniciplalities) {
+          setMunicipalities(receivedMuniciplalities);
+        }
+      } catch (err) {
+        dispatch(alertActions.error('Failed to fetch municipalities'));
+      }
+    };
+    internal();
+  }, []);
+
+  // fetch metadata types
+  useEffect(() => {
+    const internal = async () => {
+      try {
+        const res = await fetch('/api/metadataType');
+        const { status, ok } = res;
+        if (!ok) {
+          const err = new Error();
+          err.status = status;
+          throw err;
+        }
+        const receivedMetadataTypes = await res.json();
+        setMetadataTypes(receivedMetadataTypes);
+      } catch (err) {
+        dispatch(alertActions.error('Failed to fetch metadata types'));
+      }
+    };
+    internal();
+  }, []);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const res = await fetch('/api/Metadata', {
+        method: 'PUT',
+        body: JSON.stringify(state),
+        headers: {
+          'Content-Type': 'application/json',
```

Comment:
Might as well just add the Authorization header in this branch? as it is now it doesn't work as the backend requires authentication.